src/ucp/core/ucp_ep.c

```diff
@@ -1035,39 +1035,40 @@ void ucp_ep_destroy(ucp_ep_h ep)
     return;
 }
 
-static int
-ucp_ep_config_lane_is_dst_rsc_index_equal(const ucp_ep_config_key_t *key1,
-                                          ucp_lane_index_t lane1,
-                                          const ucp_ep_config_key_t *key2,
-                                          ucp_lane_index_t lane2)
+static int ucp_ep_lane_is_dst_index_equal(ucp_rsc_index_t dst_index1,
```

Comment:
ucp_ep_lane_is_dst_index_**match**
and ucp_ep_config_lane_is_peer_**match**
to distinguish from strict-equah

Reply:
done